<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>化学工具</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="plainText">
        <string/>
       </property>
       <property name="placeholderText">
        <string>请输入化学式或化学方程式，()x请把()内重复x遍,系数为1请写出      例子：H2SO4    3 Fe + 4 H2O = 1 Fe3O4 + 4 H2</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="button__shiliang">
         <property name="text">
          <string>化学式量计算</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button__peiping">
         <property name="text">
          <string>验证化学方程式配平</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
